src/mediarithmics/plugins/common/BasePlugin.ts

```diff
@@ -124,7 +124,8 @@ export class PropertiesWrapper {
 export abstract class BasePlugin {
   multiThread: boolean = false;
 
-  INSTANCE_CONTEXT_CACHE_EXPIRATION: number = 120000;
+  // Default cache is now 10 min to give some breathing to the Gateway
+  INSTANCE_CONTEXT_CACHE_EXPIRATION: number = 600000;
```

Comment:
Note: This is an open project (public repo) and we're doing the comments / commits / docs in english only ;-)

This is a good idea to stop making this configurable as we realize that it can have a direct impact on the Gateway API load. I'll add it in the comments.

However, as this changes is proposed as part of the 0.7.x release, it can't contain any breaking changes -> removing this variable or making it private would break any plugin impl. that is relying on it.

As this whole part will change in the 0.8.x releases (e.g. the next "major" update with breaking changes), I propose to deal with it then.

Ok for you?

Reply:
Fair point, LGTM !